src/commonMain/kotlin/com/carterharrison/ecdsa/EcKeyGenerator.kt

```diff
@@ -1,8 +1,8 @@
 package com.carterharrison.ecdsa
 
-import java.math.BigInteger
-import java.security.PrivateKey
-import java.security.SecureRandom
+import com.ionspin.kotlin.bignum.integer.BigInteger
```

Comment:
Why do we need to use this instead of the regular Java BigInteger? I've been out of the Java/Kotlin world for awhile.

Reply:
Sadly I need this as Kotlin standard library doesn't support BigInteger. This is needed to make the library support multiple platforms and not only JVM.

Given the importance of BigInteger in this library I thought it was better to use an external and well tested library